core/bundle/changes/changes.go

```diff
@@ -145,6 +150,8 @@ func existingOffersFromModel(ctrlModel *Model) map[string]set.Strings {
 	return existingOffers
 }
 
+var logger = loggo.GetLogger("juju.core.bundle.changes")
```

Comment:
I know we're trying to move away from this pattern, but changing all the structs and factory signatures to properly inject a logger seems overkill for something that is only a theoretical possibility, and will be dropped in Juju 4 anyway